backend/src/main/java/codezap/category/service/CategoryService.java

```diff
@@ -24,13 +27,18 @@ public class CategoryService {
     private final TemplateRepository templateRepository;
 
     @Transactional
-    public CreateCategoryResponse create(Member member, CreateCategoryRequest createCategoryRequest) {
-        String categoryName = createCategoryRequest.name();
-        validateDuplicatedCategory(categoryName, member);
-        Category category = categoryRepository.save(new Category(categoryName, member));
+    public CreateCategoryResponse create(Member member, CreateCategoryRequest request) {
+        validateDuplicatedCategory(request.name(), member);
+        validateOrdinal(member, request);
+
+        Category category = categoryRepository.save(createCategory(member, request));
         return CreateCategoryResponse.from(category);
     }
 
+    private Category createCategory(Member member, CreateCategoryRequest request) {
+        return new Category(request.name(), member, request.ordinal());
+    }
```

Comment:
개인적으론 이렇게 작성하는 게 나은 것 같아 제안해요!!

재사용되지도 않는 메서드를 선언하기 위해 클래스의 라인 수가 길어진다고 생각해요. 

```suggestion
    @Transactional
    public CreateCategoryResponse create(Member member, CreateCategoryRequest request) {
        validateDuplicatedCategory(request.name(), member);
        validateOrdinal(member, request);
        Category category = new Category(request.name(), member, request.ordinal());
        Category saved = categoryRepository.save(category);
        return CreateCategoryResponse.from(saved);
    }
```